src/bin/server.rs

```diff
@@ -29,9 +29,9 @@ pub async fn main() -> mini_redis::Result<()> {
     Ok(())
 }
 
-#[derive(StructOpt, Debug)]
-#[structopt(name = "mini-redis-server", version = env!("CARGO_PKG_VERSION"), author = env!("CARGO_PKG_AUTHORS"), about = "A Redis server")]
+#[derive(Parser, Debug)]
+#[clap(name = "mini-redis-server", version, author, about = "A Redis server")]
 struct Cli {
-    #[structopt(name = "port", long = "--port")]
+    #[clap(name = "port", long = "--port")]
```

Comment:
i don't think the `name` value is necessary here, since the name of the field is the same as the `name` value:
```suggestion
    #[clap(long = "--port")]
```